src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -140,6 +140,17 @@ private void sendNotificationToRespondent(CaseDetails caseDetails) {
                     );
                 }
             }
+            //send notification for applicants
+            for (Element<PartyDetails> applicant : caseData.getApplicants()) {
+                if (!StringUtils.isEmpty(applicant.getValue().getEmail())) {
+                    emailService.send(
+                        applicant.getValue().getEmail(),
+                        EmailTemplateNames.CA_CITIZEN_RES_NOTIFICATION,
+                        buildRespondentEmail(caseDetails, applicant.getValue()),
+                        LanguagePreference.english
```

Comment:
Is there anyway we will get welsh notification later? if not, please ignore.

Reply:
also, can you check the for loop and if else condition. seems respondent will be notified multiple times based on the for loop for applicant.

Reply:
@AlokDatta currently all email templates doesn't support welsh translations.
 Yes there is a way, we can below method to add language instead of making en default.
`public static LanguagePreference getLanguagePreference(CaseData caseData) {
        boolean preferredLanguageIsWelsh = Optional.ofNullable(caseData.getLanguagePreferenceWelsh())
            .map(YesOrNo.Yes::equals)
            .orElse(false);

        return preferredLanguageIsWelsh ? LanguagePreference.welsh : LanguagePreference.english;
    }`

Reply:
No, respondents will not get notified multiple times. You might have seen FL401 respondent block which is in the else block